store/src/test/java/org/apache/rocketmq/store/DefaultMessageStoreTest.java

```diff
@@ -43,15 +43,24 @@
     private SocketAddress BornHost;
     private SocketAddress StoreHost;
     private byte[] MessageBody;
+    private MessageStore messageStore;
 
     @Before
     public void init() throws Exception {
         StoreHost = new InetSocketAddress(InetAddress.getLocalHost(), 8123);
         BornHost = new InetSocketAddress(InetAddress.getByName("127.0.0.1"), 0);
+
+        messageStore = buildMessageStore();
+        boolean load = messageStore.load();
+        assertTrue(load);
+        messageStore.start();
     }
 
     @After
     public void destory() {
+        messageStore.shutdown();
+        messageStore.destroy();
```

Comment:
As i remember, method of destroy may not clean all files generated.

Reply:
Delete created files has been did before at this file after destroy.